code/src/components/Feed/Feed.module.css

```diff
@@ -0,0 +1,9 @@
+.feedGrid {
+  background-color: whitesmoke;
+  display: grid;
+  grid-template-columns: 1fr;
```

Comment:
I don't think you need grid here? or at least I could not se any differens while testing in the inspector